client/me/billing-history/upcoming-charges-table.jsx

```diff
@@ -25,20 +30,22 @@ module.exports = React.createClass( {
 			<TransactionsTable
 				transactions={ transactions }
 				initialFilter={ { date: { newest: 20 } } }
-				description={ function( transaction ) {
-					var site = this.props.sites.getSite( Number( transaction.blog_id ) );
-
-					if ( site ) {
-						return (
-							<div className="transaction-links">
-								<a href={ purchasesPaths.managePurchase( site.slug, transaction.id ) }>
-									{ this.translate( 'Manage Purchase' ) }
-								</a>
-							</div>
-						);
-					}
-				}.bind( this ) }
-			/>
+				emptyTableText={ emptyTableText }
+				transactionRenderer={ this.renderTransaction } />
 		);
+	},
+
+	renderTransaction: function( transaction ) {
+		var site = this.props.sites.getSite( Number( transaction.blog_id ) );
+
+		if ( site ) {
```

Comment:
The inverse condition probably makes more sense here to keep the level of indentation down…

```js
if ( ! site ) {
	return null;
}

return (
	<div className="transaction-links">
		…
	</div>
);
```